source/postprocess/visualization/shear_stress.cc

```diff
@@ -73,7 +73,8 @@ namespace aspect
 
             const double eta = out.viscosities[q];
 
-            const SymmetricTensor<2,dim> shear_stress = 2*eta*compressible_strain_rate;
+            // Compressive stress is positive in geoscience applications
+            const SymmetricTensor<2,dim> shear_stress = -2.*eta*compressible_strain_rate;
```

Comment:
The "compressible_strain_rate" variable is actually the incompressible part of the strain rate. Ditto for the other two files. Suggest changing the name to incompressible_strain_rate?

This is my only quibble :)

Reply:
Thanks for having a look. It depends on what the material model returns for `(this->get_material_model().is_compressible()` whether the strain rate in `compressible_strain_rate` is in fact compressible or the incompressible part. So the name is not always correct. But that's not really related to this PR, so I'll leave that for now, OK?

Reply:
Hmm, the relevant code above is:

>   = (this->get_material_model().is_compressible()
>                  ?
>                  strain_rate - 1./3 * trace(strain_rate) * unit_symmetric_tensor<dim>()
>                  :
>                  strain_rate);

Even if the material is compressible, the pressure is removed (-1/3 * trace(strain_rate)) and thus you have the deviatoric strain rate. If an incompressible formulation is used, then should the strain rate by default be deviatoric?

If the above is correct, would `deviatoric_strain_rate` be the most accurate name?

No preference on whether this is resolved now or in a follow-up PR, but will leave that up to @bobmyhill.

Reply:
@naliboff Yes, that's what I was trying to say. Thanks for clarifying!
I'm fine for the rename to be a separate PR.